_summaries/salimans2016weight_2.md

```diff
@@ -1,19 +1,64 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{salimans2016weight}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: samatharhay
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+The paper introduces weight normalization, a reparameterization of the weight vectors of a neural network, to speed up convergence of stocastic gradient descent.
+
+In standard neural networks the weighted sum computed by each neuron is as follows:
+**intsert equation 1**
+where w (weights) and x (input) are both k-dimensional vectors and b is a scalar bias term.
+
+In weight normalization each w vector is reparameterized in terms of a parameter vector v, of dimensionality k, and a scalar parameter g: 
+**insert equation 2**
+
```

Comment:
I think it would be a good idea to mention the data-dependent initialization of parameters somewhat. Since batch normalization can fix the scale of features but weight normalization can't, the authors proposed this initialization so the parameters can approximately be at the scale of the feature outputs of each layer to get zero mean and unit variance in the output of features.